### PR TITLE
Initial Docker page

### PR DESCRIPTION
**Pull request by donatieng**
Add docker user guide.

### CODE REVIEW

**Comment by donatieng** [comment]
Thanks Saheer, I cannot approve as I opened the original PR but it looks good :D

**Review by saheerb — APPROVED** [review]

File: docs/tools/docker/docker.md
Comment (ChrisGooch-Arm) [inline docs/tools/docker/docker.md:43]: is this meant to be a note?

Comment (ChrisGooch-Arm) [reply docs/tools/docker/docker.md:43]: should be "as the filesystem" (I can fix this in dev after merging)

Comment (saheerb) [reply docs/tools/docker/docker.md:43]: > is this meant to be a note?

This is meant to be a "tip".

Comment (saheerb) [reply docs/tools/docker/docker.md:43]: > should be "as the filesystem" (I can fix this in dev after merging)

Thanks, please fix as you mentioned.

**Pull request merged**
